src/openassetio-python/package/openassetio/hostApi/Manager.py

```diff
@@ -480,6 +511,23 @@ def getWithRelationships(
 
         @param context Context The calling context.
 
+        @param successCallback Callback that will be called for each
+        successful relationship query. It will be
+        given the corresponding index of the relationship in
+        `relationshipTraitsDatas` along with a list of entity references
+        for entities that have the specified relationship. If there
+        are no relations, an empty list will be supplied. The callback
```

Comment:
[Consider]
`an empty list will be supplied` seems ambiguous, you could mean the `relationshipTraitsDatas` arg has a default argument that is an empty list, or that the callback will receive an empty list as the argument.

Reply:
Added, along with a more general reshuffle [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/935/commits/d575d22518fb6a9167edd5f72ba3c57a77b8e4ea).